react/Gallery.js

```diff
@@ -1,36 +1,119 @@
 import React from 'react'
-import { PropTypes } from 'prop-types'
 import classNames from 'classnames'
+import { List, WindowScroller, AutoSizer, CellMeasurer, CellMeasurerCache } from 'react-virtualized'
+import { PropTypes } from 'prop-types'
```

Comment:
Are you sure it shouldn't be `import PropTypes from 'prop-types'` ?

Reply:
It was already there, thanks for catching it!